R/readWorkbook.R

```diff
@@ -205,3 +199,24 @@ read_sheet_names <- function(file) {
 
   return(sheets)
 }
+
+check_xlsx_path <- function(x, warn = TRUE) {
+  stopifnot(
+    is.character(x),
+    length(x) == 1,
+    file.exists(x)
+  )
+
+  bad <- stringi::stri_detect_regex(x, "\\.xls[mb]?$", case_insensitive = TRUE)
```

Comment:
We can read `xlsm` and we even have a couple of test cases for this. `xlsm` is just the default open xml format with additional macro variables enabled.

Reply:
A quick check told me that we read `xltx` (templates) as well and most likely `xltm` too.